utils/async/AsyncUtils.ts

```diff
@@ -55,7 +55,7 @@ export async function whileAsync(
     const {waitAfter = 50, waitFor = 0} = opts ?? {};
 
     // Fallback to basic loop when doc hidden: no user benefit, and throttling causes outsize waits
```

Comment:
why the inconsistency here?  I would expect this would also us XH.pageIsVisible to take advantage of the debouncing there.

Reply:
Had asked the same of @jskupsik and he was worried that there might be some extra performance overhead evaluating that getter, but I don't see how there would be.  I would also vote for using !pageIsVisible